src/ExternalSearch.Providers.Bregg/BrregExternalSearchProvider.cs

```diff
@@ -292,15 +294,19 @@ private CodeOrigin GetCodeOrigin()
             return CodeOrigin.CluedIn.CreateSpecific("brreg");
         }
 
-        public void PopulateMetadata(IEntityMetadata metadata, IExternalSearchQueryResult<BrregOrganization> resultItem, IExternalSearchRequest request)
+        public void PopulateMetadata(IEntityMetadata metadata, IExternalSearchQueryResult<BrregOrganization> resultItem, IExternalSearchRequest request, IDictionary<string, object> config)
         {
-            var code = GetOriginEntityCode(resultItem, request);
+            var jobData = new BrregExternalSearchJobData(config);
+            var code = jobData.SkipEntityCodeCreation ? request.EntityMetaData.OriginEntityCode : GetOriginEntityCode(resultItem, request);
 
             metadata.EntityType       = request.EntityMetaData.EntityType;
             metadata.Name             = request.EntityMetaData.Name;
             metadata.OriginEntityCode = code;
 
-            metadata.Codes.Add(code);
+            if (!jobData.SkipEntityCodeCreation)
+            {
+                metadata.Codes.Add(code);
```

Comment:
can be updated to `metadata.Codes.Add(GetOriginEntityCode(resultItem, request));`

Reply:
done update